### PR TITLE
Add translation for basic signatures

### PR DESCRIPTION
**Pull request by RuomeiYan**
org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java: A data structure that stores the data needed for generating python translation
org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/transform/CoreDashToPython.java: The translator that handles translation from core Dash to Python
org.alloytools.alloy.application/src/main/java/templates/VelocityTemplateFile.vm: The template I am using currently

### CODE REVIEW

File: org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    /**
+     * Constructs a new DashPythonTranslation object
+     *
+     * @param dashModule - the DashModule we want to translate
+     */
+    public DashPythonTranslation(DashModule dashModule) {
+        this.dashModule = dashModule;
+        this.basicSigLabels = dashModule.sigs.values().stream()
+                .filter(this::isSubSig)
```
Comment (Simonliuwaterloo) [inline org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java:29]: Question: what is subsig?
I saw some explanation in code
```
    /**
     * Nonnull if this sig is a PrimSig and therefore not a SubsetSig.
     */
    public final Pos             isSubsig;
```
kinda confused

Comment (RuomeiYan) [reply org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java:29]: I don't know, I also saw that explanation. I am just using this as the helper function name for now. Maybe later after we add more kinds of signatures, we could found a standardized way of naming all of them?

Comment (Simonliuwaterloo) [reply org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java:29]: I see, so you are just using this to filter out signatures outside of the scope of this ticket?

Comment (RuomeiYan) [reply org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java:29]: Yes, I could rename it to isBasicSig, but the concept of basic signature is also kind of vague. I expect the code to evolve in later tickets.

Comment (Simonliuwaterloo) [reply org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java:29]: Makes sense.

**Review by Chestshire — APPROVED** [review]

**Pull request merged**